Pavan_Kumar/SCRUM-3-find-second-largest-element-in-an-array.py

```diff
@@ -0,0 +1,12 @@
+size = int(input("Enter the number of element: "))
+arr=[]
+print("Enter",size,"numbers:")
+for i in range (0,size):
+    ele= int(input())
+    arr.append(ele)
+arr.sort()
```

Comment:
Edge Cases are not pass like if all the elements are the same of the array. Please correct this